src/ApiManagementManagement/packages.config

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Hyak.Common" version="1.0.2" targetFramework="net45" />
-  <package id="Microsoft.Azure.Common" version="2.0.3" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common" version="2.0.4" targetFramework="net45" />
```

Comment:
If you are going out with the //build or ignite release, you will need to update this to 2.1.0